src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  static newAlive() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The method `newAlive` is not optimally named as it does not create new life but filters out dead animals. Consider renaming it to more accurately reflect its functionality, such as `removeDead`.